lms/djangoapps/verify_student/models.py

```diff
@@ -588,16 +588,23 @@ class SoftwareSecurePhotoVerification(PhotoVerification):
     photo_id_key = models.TextField(max_length=1024)
 
     IMAGE_LINK_DURATION = 5 * 60 * 60 * 24  # 5 days in seconds
+    copy_id_photo_from = models.ForeignKey("self", null=True, blank=True)
```

Comment:
@cpennington I am now setting this field in method `submit` if parameter `copy_id_photo_from` is provided (which we provide for all ICRV submissions)